molecule/tests/variables/asserts/wal_g_cron_jobs.yml

```diff
@@ -24,10 +24,10 @@
   run_once: true
   ansible.builtin.assert:
     that:
-      - wal_g_backup_command[0] == "[ $(curl -s -o /dev/null -w '%{http_code}' http://{{ inventory_hostname }}:{{ patroni_restapi_port }}) = '200' ]"
-      - wal_g_backup_command[1] == " && wal-g backup-push {{ postgresql_data_dir }} > {{ postgresql_log_dir }}/walg_backup.log 2>&1"
-      - wal_g_delete_command[0] == "[ $(curl -s -o /dev/null -w '%{http_code}' http://{{ inventory_hostname }}:{{ patroni_restapi_port }}) = '200' ]"
-      - wal_g_delete_command[1] == " && wal-g delete retain FULL 4 --confirm > {{ postgresql_log_dir }}/walg_delete.log 2>&1"
+      - wal_g_backup_command[0] == "curl -I -s http://{{ inventory_hostname }}:{{ patroni_restapi_port }} | grep 200"
+      - wal_g_backup_command[1] == " && wal-g --config {{ postgresql_home_dir }}/.walg.json backup-push {{ postgresql_data_dir }} > {{ postgresql_log_dir }}/walg_backup.log 2>&1"
```

Comment:
`.walg.json` is located in the postgres user's HOME and is used by default without explicitly specifying the --config option, so this is unnecessary.

Reply:
Agree.

Reply:
fixed.